frontend/components/Button/index.tsx

```diff
@@ -0,0 +1,73 @@
+import styled, {css} from "styled-components";
+
+/**
+ * Button interface
+ * @property {varientColor} string Color of the button
+ * @property {varient} string Button type whether it's a link, solid or outline
+ * @property {?size} ButtonSizes size of the button
```

Comment:
What does {?type} mean in JSDoc? I'm not familiar with this + it needs a return between the description & properties XD

Reply:
haha May b i got it wrong. {type} variable I did it opposite. XD

Reply:
ok done.